images/win/scripts/Installers/Install-AzureModules.ps1

```diff
@@ -36,6 +36,46 @@ foreach ($module in $modules)
         }
     }
 
+    if($null -ne $module.url)
+    {
+        $assets = Invoke-RestMethod $module.url
```

Comment:
@maxim-lobanov don't we need to auth the request here with GH token to avoid rate-limiting as well as below while making use of `asset.download_url` while downloading?

Reply:
Nope, it is not needed here because we run image-generation on self-hosted machines and they don't meet rate limits (at least, we interact with GH API a lot and never faced it yet :) )